matminer/featurizers/composition.py

```diff
@@ -157,6 +158,54 @@ def implementors(self):
         return ["Jiming Chen", "Logan Ward", "Anubhav Jain"]
 
 
+class AtomicOrbitals(BaseFeaturizer):
+    '''
+    class to determine the highest occupied molecular orbital (HOMO) and
+    lowest unocupied molecular orbital LUMO in a composition. The atomic 
+    orbital energies of neutral ions with LDA DFT were computed by NIST.
+    https://www.nist.gov/pml/data/atomic-reference-data-electronic-structure-calculations
+    '''
+
+    def __init__(self):
+        pass
+
+    def featurize(self, comp):
+        '''
+        Args:
+            comp: (Composition)
+                pymatgen Composition object
+
+        Returns:
+            HOMO_character: (str)
+            HOMO_element: (Element)
+            HOMO_energy: (float in eV)
+            LUMO_character: (str)
+            LUMO_element: (Element)
+            LUMO_energy: (float in eV)
+            gap: (float in eV)
+                the estimated bandgap from HOMO and LUMO energeis
+        '''
+
+        string_comp = comp.reduced_formula
+
+        HOMO_LUMO = MolecularOrbitals(string_comp).band_edges
+
+        self.feat = {}
```

Comment:
don't use a plain dict for this. The order of keys() and values() can change in a dict. It will be consistent 99% of the time so you won't typically notice it, but it *does* change.

You can either use two lists (one for the feature names, one for the feature values) or you can use an OrderedDict.

As it stands, the code will potentially return the keys and values in different orders leading to bad values